storage/pebble/testutil.go

```diff
@@ -30,3 +32,30 @@ func NewBootstrappedRegistersWithPathForTest(tb testing.TB, dir string, first, l
 	require.NoError(tb, db.Set(latestHeightKey, encodedUint64(latest), nil))
 	return db
 }
+
+func RunWithRegistersStorageWithInitialData(
+	tb testing.TB,
+	data map[uint64]flow.RegisterEntries,
+	f func(db *pebble.DB)) {
+	unittest.RunWithTempDir(tb, func(dir string) {
+		db := NewBootstrappedRegistersWithPathForTest(tb, dir, uint64(0), uint64(0))
+		r, err := NewRegisters(db, 5)
+		require.NoError(tb, err)
+
+		keys := make([]uint64, 0, len(data))
+		for k := range data {
+			keys = append(keys, k)
+		}
+		sort.Slice(keys, func(i, j int) bool { return keys[i] < keys[j] })
```

Comment:
It should not matter in which order we store the keys in pebble. Pebble is supposed to store keys in a sorted order for iteration.

And we also don't require the call to sort registers before storing them. 
```suggestion
```

Reply:
Right, `Store` requires to be called from low height to higher height.